src/Form/Form.js

```diff
@@ -386,16 +387,36 @@ class Form {
         }
     }
 
+    /**
+     * The function looks for form's control elements, and returns them if they're iterable.
+     * @param {*} selector
+     */
+    getFormControlElements(selector) {
+        // Some sites seem to be overriding `form.elements`, so we need to check if it's still iterable.
+        if (this.form instanceof HTMLFormElement && this.form.elements != null && Symbol.iterator in Object(this.form.elements)) {
+            // For form elements we use .elements to catch fields outside the form itself using the form attribute.
+            // It also catches all elements when the markup is broken.
+            // We use .filter to avoid specific types of elements.
+            const formControls = [...this.form.elements].filter((el) => el.matches(selector));
+            return [...formControls];
+        } else {
+            return null;
+        }
+    }
+
```

Comment:
The function now has a clear separate purpose, that's different from querying the elements which is what we rather do in most cases.